src/codemodder/dependency_management/dependency_manager.py

```diff
@@ -27,6 +28,10 @@ def write(
                 return RequirementsTxtWriter(
                     self.dependencies_store, self.parent_directory
                 ).write(dependencies, dry_run)
+            case "pyproject.toml":
```

Comment:
I feel pretty strongly that these kinds of values should be represented by an enum. The fact that it is being used as cases for `match` really emphasizes that to me.